src/_includes/banner.html

```diff
@@ -1,9 +1,8 @@
 <div class="banner">
   <p class="banner__text">
-    Dart 3 is here with records and patterns, alongside Google I/O!
-    <br>
-    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Check out the blog post</a> and 
-    <a href="https://io.google/2023/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_banner&utm_content="
-    class="no-automatic-external">register to watch</a> now.
+    Dart 3 is here, with 100% sound null safety, records, and patterns!
+    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Learn more</a><br>
+    Learn how Dart and Flutter are both racing forward at Google I/O 2023!
+    <a href="https://io.google/2023/program/?q=keynote,technical-session,workshop,flutter" class="no-automatic-external">Watch now</a>
```

Comment:
**suggestion:** Headlines can be a little choppy. That's OK. Shortening the text.

```suggestion
    Dart 3: Now with 100% sound null safety, records, and patterns!
    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Learn more</a><br>
    Racing Forward at I/O 2023 with Flutter and Dart
    <a href="https://io.google/2023/program/?q=keynote,technical-session,workshop,flutter" class="no-automatic-external">Watch the videos!</a>
```